tests/quick_mode_removal_test.py

```
@@ -0,0 +1,300 @@
+    """Integration tests for various user scenarios."""
+    
+    @pytest.mark.integration
+    @pytest.mark.asyncio
+    async def test_evolution_with_real_api(self):
+        """Test evolution with real Google API key."""
+        import os
+        if not os.getenv("GOOGLE_API_KEY"):
+            pytest.skip("GOOGLE_API_KEY not available")
+            
+        # Test with minimal settings
+        await _run_evolution_pipeline(
```

Ruff (0.12.2)

Undefined name `_run_evolution_pipeline`

(F821)